ansible/testbed-cli.sh

```diff
@@ -225,7 +226,8 @@ function stop_topo_vms
 
   echo "Stopping VMs for testbed '${testbed_name}' on server '${server}'"
 
-  ANSIBLE_SCP_IF_SSH=y ansible-playbook -i $vmfile testbed_stop_VMs.yml --vault-password-file="${passwd}" -l "${server}" -e VM_base="$vm_base" -e topo="$topo" $@
+  ANSIBLE_SCP_IF_SSH=y ansible-playbook -i $vmfile testbed_stop_VMs.yml --vault-password-file="${passwd}" -l "${server}" \
+	  -e VM_base="$vm_base" -e vm_type="$vm_type" -e topo="$topo" $@
```

Comment:
Skip "add-topo-vms" if "vm_type=='ceos'"?

Reply:
I didn't notice that the code for skipping ceos in add-topo-vms and stop-topo-vms is already there.

Reply:
The logic had existed in the beginning